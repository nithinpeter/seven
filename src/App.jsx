import React from 'react';
import styled from 'styled-components';
import tile from './tile.jpg';
import logo from './logo.png';

const App = styled.div`
  height: 100vh;
  width: 100vw;
  background-image: url(${tile});
  background-repeat: no-repeat;
  background-size: cover;
  position: relative;
`;

const Seven = styled.div`
  height: 33%;
  width: 100%;
  display: flex;
  align-items: center;
  position: absolute;
  bottom: 0;
  left: 0;
  background: rgba(0, 0, 0, 0.4);
  img {
    padding: 0 3vmin;
    max-height: 25vmin;
  }
  h1 {
    font-size: 10vmin;
    margin: 0;
    color: #ffffff;
  }
`;

export default () => (
  <App>
    <Seven>
      <img src={logo} alt="seven logo" />
      <h1>Home and Away</h1>
    </Seven>
  </App>
);
