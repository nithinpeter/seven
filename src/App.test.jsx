import React from 'react';
import ReactDOM from 'react-dom';
import ReactDOMServer from 'react-dom/server';
import App from './App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders without crashing', () => {
  expect(ReactDOMServer.renderToString(<App />)).toMatchSnapshot();
});
