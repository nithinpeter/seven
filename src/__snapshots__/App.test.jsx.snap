// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `"<div class=\\"sc-bdVaJa bwOeS\\" data-reactroot=\\"\\"><div class=\\"sc-bwzfXH fllVrW\\"><img src=\\"logo.png\\" alt=\\"seven logo\\"/><h1>Home and Away</h1></div></div>"`;
